src/plugins/expressions/common/service/expressions_services.ts

```diff
@@ -58,6 +61,13 @@ export interface ExpressionExecutionParams {
    */
   debug?: boolean;
 
+  /**
+   * Makes a `KibanaRequest` object available to expression functions. Useful for
+   * functions which are running on the server and need to perform operations that
+   * are scoped to a specific user.
+   */
+  kibanaRequest?: KibanaRequest;
```

Comment:
can we rather expose a getter ? exposing kibanaRequest directly allows for modifying of it from inside expressions which we don't want to allow.

Reply:
agreed, will update